app/test/unit/stores/updates/update-changed-files-test.ts

```diff
@@ -31,7 +33,7 @@ const files = [
 ]
 
 describe('updateChangedFiles', () => {
-  describe('workingDirectory', () => {
+  describe(ChangesSelectionKind.WorkingDirectory, () => {
```

Comment:
💭 any interest in adding a corresponding stash test in the scope of this PR? Feel free to punt to another issue if you're feeling under pressure..

Reply:
lol, I think this is a search and replace gone awry. Let me just reset that. Stashing doesn't have anything to do with `updateChangedFiles`